aldjemy/orm.py

```diff
@@ -88,17 +88,22 @@ def _extract_model_attrs(metadata, model, sa_models):
             sec_table = tables[sec_table_qualname]
             sec_column = fk.m2m_column_name()
             p_sec_column = fk.m2m_reverse_name()
+            overlaps = (
+                fk.m2m_field_name(),
+                fk.m2m_reverse_field_name(),
+                fk.remote_field.field.get_attname(),
+                fk.remote_field.through._meta.get_field(
+                    fk.remote_field.field.m2m_field_name()
+                ).remote_field.related_name,
+                fk.remote_field.through._meta.get_field(
+                    fk.m2m_reverse_field_name()
+                ).remote_field.related_name,
+            )
```

Comment:
for my sanity, put these in a list instead of a tuple, please, because the items in the list are of the same type, and we can add items without needing to change any type signatures. Tuples are for fixed-length types that may have different types in different positions.